doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -82,12 +82,12 @@ IAM role-based operator deployment
 Before you can deploy your operator using an IAM role, associate an OpenID Connect (OIDC) provider with your role to
 authenticate with the IAM service.
 
-Associate an OpenID Connect Provider to Your Instance
+Create an OpenID Connect Provider to Your Instance
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Create an OIDC identity provider for your cluster. If your
 cluster is managed by EKS, then your cluster will already have an OIDC
```

Comment:
It is confusing to say we are creating an OIDC provider while at the same time we say one is already "attached to it". I know the eksctl docs are inconsistent on the issue, but we should be consistent so that customers are not confused. Can address now or when we revisit this as part of Arun's feedback.

Reply:
This is confusing for me as well I think for EKS 1.13 version they had to create and associate and later version they just need to associate. I can double check.